ui/app/pages/create-account/connect-hardware/index.scss

```diff
@@ -131,7 +131,7 @@
     text-align: center;
   }
 
-  &__link {
+  &-link {
```

Comment:
Huh - I think this is an unused classname :thinking: `hw-connect-link` isn't used anywhere.

Reply:
++